### ISSUE TITLE
Nullifier Visuals

### ISSUE THREAD

**Issue by Thiana** [maruohon/enderutilities#79]
Could you make the visual of the Nullifier match more closely to regular block size? At the moment it's large enough to block a significant portion of the screen. In the image below, the standard size is on the left and the Nullifier on the right. Thank you.

![2018-08-02_00 36 42](https://user-images.githubusercontent.com/405832/43584354-3153e828-9630-11e8-974b-54848337024a.png)

**Comment by maruohon** [comment]
Whaaat?! O_o
This is how it has always been supposed to look like:
![2018-08-02_15 55 45](https://user-images.githubusercontent.com/2331050/43584996-a32f9cc6-966c-11e8-84b1-f6147faecacf.png)

So some kind of cross-mod issue, and with rendering nonetheless ... >_> Two of my worst arch enemies...

**Comment by Thiana** [comment]
I'm not sure this actually counts as a bug since It looks like it's Foamfix by @asiekierka causing the issue. Specifically the optimization:

```
# Remove unnecessary data from a pointlessly cached ModelLoader instance. [default: true]
B:modelLoaderCleanup=false
```

Maybe asie can suggest a way to have foamfix ignore the Nullifier instance?

Versions:
forge 14.23.4.2739
enderutilities-1.12.2-0.7.11.jar
foamfix-0.9.9.1-1.12.2-anarchy.jar

With foamfix defaults:

![2018-08-03_06 48 49](https://user-images.githubusercontent.com/405832/43639813-7dcff5f0-96eb-11e8-8bff-6cf71635c631.png)


With B:modelLoaderCleanup disabled.

![2018-08-03_06 49 50](https://user-images.githubusercontent.com/405832/43639821-83d42ebc-96eb-11e8-8d45-cf4440bbd3c0.png)

**Comment by Thiana** [comment]
After using the Nullifier for a few weeks I agree that it's a touch too large. Could you consider shrinking it slightly and lowering it a bit in future versions? Thanks.

PS: Since it looks like asie doesnt follow mentions, would you like me to open an issue in the foamfix repo about B:modelLoaderCleanup or just close this issue?

**Comment by maruohon** [comment]
Yeah I'll try to adjust it for the next release. If you want to try to follow up on this on the Foamfix side, go ahead. I don't know if there is something I should be doing differently on my side, so I'd be glad to hear that too if it's the case.

**Comment by asiekierka** [comment]
I do! I actually looked at this issue, but couldn't find anything immediately wrong.
